src/widgets/helper/ChannelView.cpp

```diff
@@ -1786,8 +1789,48 @@ void ChannelView::leaveEvent(QEvent * /*event*/)
     this->unpause(PauseReason::Mouse);
 }
 
+bool ChannelView::event(QEvent *event)
+{
+     if (event->type() == QEvent::Gesture)
+     {
+         return gestureEvent(dynamic_cast<QGestureEvent*>(event));
+     }
+
+    return BaseWidget::event(event);
+}
+
+bool ChannelView::gestureEvent(const QGestureEvent *event)
+{
+     if (QGesture *pan = event->gesture(Qt::PanGesture))
+     {
+         auto const *gesture = dynamic_cast<QPanGesture *>(pan);
```

Comment:
```suggestion
         const auto *gesture = dynamic_cast<QPanGesture *>(pan);
```

additionally, you should probably sanity-check that `gesture` is not a `nullptr` since we've used `dynamic_cast`